tests/integration/api/test_decision_environment.py

```diff
@@ -97,6 +97,101 @@ def test_create_decision_environment(
         assert status_message in response.data["eda_credential_id"]
 
 
+@pytest.mark.parametrize(
+    ("return_code", "image_url", "unallowed"),
+    [
+        (status.HTTP_201_CREATED, "registry.com/group/img1:tag", ""),
+        (status.HTTP_201_CREATED, "registry.com/group/img1", ""),
+        (
+            status.HTTP_400_BAD_REQUEST,
+            "http://registry.com/img1:tag1",
+            "scheme not allowed",
+        ),
+        (
+            status.HTTP_400_BAD_REQUEST,
+            "//registry.com/img1:tag1",
+            "network location not allowed",
+        ),
+        (
+            status.HTTP_400_BAD_REQUEST,
+            "registry.com/img1:tag;parameter",
+            "parameters not allowed",
+        ),
+        (
+            status.HTTP_400_BAD_REQUEST,
+            "registry.com/img1:tag1?query",
+            "query not allowed",
+        ),
+        (
+            status.HTTP_400_BAD_REQUEST,
+            "registry.com/img1:tag#fragment",
+            "fragment not allowed",
+        ),
+        (
+            status.HTTP_400_BAD_REQUEST,
+            "registry.com/group/:tag",
+            "name does not match OCI standard",
+        ),
+        (
+            status.HTTP_400_BAD_REQUEST,
+            "registry.com/group/img1:",
+            "tag does not match OCI standard",
+        ),
+        (
+            status.HTTP_400_BAD_REQUEST,
+            "registry.com/group/bad^img1",
+            "name does not match OCI standard",
+        ),
+        (
+            status.HTTP_400_BAD_REQUEST,
+            "registry.com/group/img1:bad^tag",
```

Comment:
@jshimkus-rh Can this include the port number. registry.com:8443/group1/img1?

Reply:
Good point.  I thought the name was the totality but it excludes the host:port.  Gonna have to make some changes.